java/vector/src/test/java/org/apache/arrow/vector/TestVarCharViewVector.java

```diff
@@ -371,6 +373,67 @@ public void testMixedAllocation() {
     }
   }
 
+  @Test
+  public void testSetNullableViewVarCharHolder() {
+    try (final ViewVarCharVector viewVarCharVector = new ViewVarCharVector("myvector", allocator)) {
+      viewVarCharVector.allocateNew(0, 0);
+      final List<byte[]> strings = List.of(STR0, STR1, STR2, STR3, STR4, STR5);
+
+      NullableViewVarCharHolder stringHolder = new NullableViewVarCharHolder();
+
+      setAndCheck(viewVarCharVector, 1, strings.get(0), stringHolder);
```

Comment:
To check overwrite?

Reply:
Why do we need this? Could you explain a bit?

Reply:
I think it's not needed. It's just missed it while adjusting the code. But you reminded me that it is necessary to add some overwrite test cases.

Reply:
I don't think it is a must. It is okay to leave it at this moment.